shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/conf/ct/SyTableReplications.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.replicator.exec.conf.ct;
+
+import java.util.List;
+
+import javax.validation.Valid;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+public class SyTableReplications {
+
+  private @Valid @NotEmpty List<SyTableReplication> tableReplications;
```

Comment:
Can be empty that's ok, that means don't override anything right?

Reply:
Then why even specify this tag in the config ? If the tag is specified then it needs to be completed.

Reply:
Ok this means you can't say:
```
table-replications:
```
Right ok, I thought it meant you have to specify at least one table-replication. I'm ok if we keep it also don't mind if it goes don't see the problem with above in the the yml but perhaps I'm still missing something